docs/guides/overview-extensibility-features.md

```diff
@@ -0,0 +1,63 @@
+---
+id: overview-extensibility-features
+title: Overview of available extensibility features
+sidebar_label: Extensibility features overview
+slug: /overview/all-features
+---
+
+This page provides an overview of all of the extensibility features that are available that can help you develop what you need to on top of and alongside DHIS2.
+
+-   API: The whole [Web REST API](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/introduction.html) is available to use. Operations on data and metadata are generic and available to be used by any extension.
```

Comment:
suggestion: similar to "integrations" header, maybe the first part can be divided under two headers, one for "core" (API, routes etc..) and the other for "custom apps" (with apps, platform, plugins...)


or the headers can be "core extension points", "apps extension points", "integrations" etc...